test/MusicStore.E2ETests/PublishAndRunTests.cs

```diff
@@ -18,6 +18,8 @@ public static TestMatrix TestVariants
             => TestMatrix.ForServers(ServerType.IISExpress, ServerType.Kestrel, ServerType.HttpSys)
                 .WithTfms(Tfm.NetCoreApp22, Tfm.NetCoreApp21, Tfm.NetCoreApp20, Tfm.Net461)
                 .WithAllApplicationTypes()
+                .WithAllAncmVersions()
+                .WithAllHostingModels()
```

Comment:
How do you filter out ANCM inproc with netcoreapp2.0 here?

Reply:
I added it here: https://github.com/aspnet/Hosting/pull/1436/files#diff-ff26c4beb20e3d2bc16da63569390fd8